src/App.scss

```diff
@@ -81,27 +102,27 @@ $timelineCellLabelColor: yellow;
    #timeline-track-area {
       grid-row: 3;
       grid-column: 1/3;
-	  display: flex;
-	  flex-direction: column;
+      display: flex;
+      flex-direction: column;
 
-	  #timeline-track-main-area {
-		  display: flex;
-		  align-items: center;
-	  }
+      #timeline-track-main-area {
+         display: flex;
+         align-items: center;
+      }
```

Comment:
I guess my IDE auto-formatted this tabbed whitespace to match the rest of the file when I copy-pasted my new #error-popup styling